modules/wallet.go

```diff
@@ -364,8 +364,8 @@ type (
 		RegisterTransaction(t types.Transaction, parents []types.Transaction) TransactionBuilder
 
 		// Rescanning reports whether the wallet is currently rescanning the
-		// blockchain.
-		Rescanning() bool
+		// blockchain, and if so, the highest blockheight scanned.
+		Rescanning() (bool, types.BlockHeight)
```

Comment:
(blockheight, bool) would be my preference, though the name doesn't quite fit in that case. Maybe change to RescanProgress ?